### PR TITLE
Feature 1 doc

### PR DESCRIPTION
**Pull request by josepvilabadillo**
Added a README.md and a CLIUsage.md to the repository.

### CODE REVIEW

File: CLIUsage.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    pubKeyFile (only base64 format is accepted for the public key)
+    privKeyFile (in openSSH format)
+    serverFingerPrint1 (e.g.: 11:22:33:44:55:66:77:88:99:00:aa:bb:cc:dd:ee:ff:8c:c7:4a:f5:42:a6:4b:64:07:6b:03:ec:c8:0a:ab:9e)
+    {serverFingerPrint2} (optional)
+    {serverFingerPrint3} (optional)
+    {...}
+
+
+## Return values
+* 0 if anything goes fine
```
Comment (Blackvals) [inline CLIUsage.md]: If _everything_ goes fine

File: README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Renames the server file srcFile to dstFile
+#### Parameters
+* **srcFile**: Path to the file to be renamed in the SFTP server
+* **dstFile**: New name of the file 
+#### Exceptions
+* **InvalidConnectionException**: The Connection object is not connected (the isConnected() method returns false)
+* **SFTPCannotRenameRemoteFileException**: The file could not be renamed in the server
+---
+
+    virtual void close()
```
Comment (Blackvals) [inline README.md]: Why the virtual here and not on the others? Probably not relevant for usage documentation

File: README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Closes the connection. After calling this function any call to isConnected() will return false.
+
+---
+
+## SftpLibsshAdapter
+The implementation using libssh, implements these two interfaces through the SFTPClient and SFTPConnection classes.
```
Comment (Blackvals) [inline README.md]: Maybe we could add a link to the libssh.org page

**Review by Blackvals — APPROVED** [review]

File: CLIUsage.md (hunk 1 above)
Comment (josepvilabadillo) [reply CLIUsage.md]: Right, changed.

File: README.md (hunk 2 above)
Comment (josepvilabadillo) [reply README.md]: Forgot to remove it when doing copy paste. Fixed.

**Pull request merged**